core/lib/dal/migrations/20240308203309_create_soft_pruning_table.up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE pruning_info
+(
+    last_soft_pruned_l1_batch  BIGINT,
+    last_soft_pruned_miniblock BIGINT,
+    last_hard_pruned_l1_batch  BIGINT,
+    last_hard_pruned_miniblock BIGINT
```

Comment:
It would require one less query, but the existing queries would be a bit more convoluted. I think it boils down to a discussion whether it's okay to have some de-normalized columns. I like having those explicit columns as it makes this component more isolated from rest of the system and to reason about invariants of it, we also keep information when the batch was hard pruned there (in theory we could also use prune_log for that, but I'm not sure yet, we'd like to keep it longer term).